### ISSUE TITLE
global name 'URLError' is not defined

### ISSUE THREAD

**Issue by dcarthy** [10se1ucgo/DisableWinTracking#350]
**INFORMATION:** When trying to run the extracted exe after latest Windows 10 updates 13-07-2017

[screenshot error](http://i.imgur.com/oHZK1PL.png)
[screenshot updates](http://i.imgur.com/AtxsP9S.png)

* Windows 10 (Pro):
* 64bit:
* Options used: none

**LOG:**

```15:32:16 INFO: Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
15:32:16 INFO: ('Windows', 'David', '10', '10.0.14393', 'AMD64', 'Intel64 Family 6 Model 60 Stepping 3, GenuineIntel')
15:32:16 INFO: DisableWinTracking version 3.1.2
15:32:17 CRITICAL: Traceback (most recent call last):
  File "dwt.py", line 486, in <module>
  File "dwt_about.py", line 172, in update_check
NameError: global name 'URLError' is not defined

```

Hope you can help!

Cheers

**Comment by Frodgy** [comment]
I had the same problem on clean Windows 10 before I installed all drivers and updates.

**Comment by soiax** [comment]
I had the same, after visiting github.com, it was working, in an other issue someone said it has to download githubs ssl cert first. It checks for updates on github.com

**Comment by sdfg2** [comment]
Yeah, that needs to be fixed.  I don't want to even turn on my network adapter until I've got rid of all the crap.

**Comment by brown0gx** [comment]
![image](https://user-images.githubusercontent.com/31367150/29740560-989674f6-8a9c-11e7-84c1-5e5e13951aa4.png)   - What is the switch to disable checking online?   Can this be made available so we can run the program without requiring a internet link?   - Thankyou for your assistance

**Comment by Ruined1** [comment]
Fixed in version 3.1.3
